spec/features/login_spec.rb

```diff
@@ -60,17 +60,17 @@
     it 'e-mail has already been taken' do
       register('test@test.com', '12345', '12345')
       register('test@test.com', '12345', '12345')
-      expect(page).to have_content 'has already been taken'
+      expect(page).to have_content 'Не уникальное значение.'
     end
 
     it 'password is too short' do
       register('test@test.com', '1', '12345')
-      expect(page).to have_content 'is too short (minimum is 3 characters)'
+      expect(page).to have_content 'Короткое значение.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.